mmv1/third_party/terraform/resources/resource_bigtable_instance.go

```diff
@@ -107,6 +107,12 @@ func resourceBigtableInstance() *schema.Resource {
 										Required:    true,
 										Description: `The target CPU utilization for autoscaling. Value must be between 10 and 80.`,
 									},
+									"storage_target": {
+										Type:        schema.TypeInt,
+										Optional:    true,
+										Computed:    true,
+										Description: `The target storage utilization for autoscaling, in GB, for each node in a cluster. This number is limited between 2560 (2.5TiB) and 5120 (5TiB) for a SSD cluster and between 8192 (8TiB) and 16384 (16 TiB) for an HDD cluster. If not set, whatever is already set for the cluster will not change, or if the cluster is just being created, it will use the default value of 2560 for SSD clusters and 8192 for HDD clusters.`,
```

Comment:
nit: Is "between" inclusive?

Reply:
Yes. And just following the other descriptions above.